README.md

```diff
@@ -144,8 +144,8 @@ You can deploy this project using:
 
 > Describe 1 - 3 features you will add to the project.
```

Comment:
- Hey, please, let's remove this placeholder text; the other images of other placeholder text will be added in the review below. These placeholder texts were added as a guide for you to follow in updating your readme by adding the appropriate content to their respective section of the readme. As such, they are meant to be removed after you've made the update to your readme.

:point_down: 
![Screenshot from 2023-03-01 12-30-49](https://user-images.githubusercontent.com/59397159/222127511-b406aab9-2adb-408d-92bd-d2cd91166383.png)
![Screenshot from 2023-03-01 12-30-42](https://user-images.githubusercontent.com/59397159/222127513-6ca53aad-72ea-46e3-9d3e-e236288e4815.png)
![Screenshot from 2023-03-01 12-30-32](https://user-images.githubusercontent.com/59397159/222127515-5201210b-c493-4982-bab2-db6cfcc4c824.png)
![Screenshot from 2023-03-01 12-30-21](https://user-images.githubusercontent.com/59397159/222127517-34b88f6f-3990-42a0-af9e-6e8a20ac9c8f.png)
![Screenshot from 2023-03-01 12-30-10](https://user-images.githubusercontent.com/59397159/222127519-8c212db3-6d16-441d-be68-32a7cb6f0e02.png)
![Screenshot from 2023-03-01 12-29-41](https://user-images.githubusercontent.com/59397159/222127522-2ba25284-059e-4734-b738-26d1534a967c.png) :point_up_2: 

---

### REMOVE THE BROKEN LINK AND EMPTY DEPLOYMENT SECTION 

-  [ ] Hey, since you've not deployed the app, please kindly remove the broken links to the live demo and the employ deployment section to keep the readme more professional. Aslo, remove them from the navigation list of the readme.
:point_down: 
![Screenshot from 2023-03-01 12-33-48](https://user-images.githubusercontent.com/59397159/222128546-1f5c9294-60df-4a81-9f1e-f0cf5a9ac2c5.png)
![Screenshot from 2023-03-01 12-33-30](https://user-images.githubusercontent.com/59397159/222128548-78df333c-ae4c-4660-8ab7-e37a5e83e1a5.png) :point_up_2: 

---

### REMOVE THE OPTIONAL AND UNUSED SECTION 

- [ ] Hey, as the FAG section is OPTIONAL, please, kindly remove it from the readme and navigation also. Should you want to add it, then add some questions and answers users could possibly want to ask about the app, as this would make your readme more professional.
- [ ] Also, since you didn't use any database or server, kindly remove it also.

:point_down: 
![Screenshot from 2023-03-01 12-40-29](https://user-images.githubusercontent.com/59397159/222129383-e60149e4-89d0-4652-9d2c-01978100eebf.png)
![Screenshot from 2023-03-01 12-37-38](https://user-images.githubusercontent.com/59397159/222129388-ca34ae9e-eb39-4448-95b0-d6c928ff39f2.png):point_up_2: 

--- 


## Show your support

- [ ] Please, kindly update the text content to something like `If you like this project give it a star` :star: to make the text more descriptive compared to what you have now. :pray: